lib/JITServer/JITServer.cpp

```diff
@@ -131,7 +131,13 @@ ServerInitializeThreadContext(
 {
     AUTO_NESTED_HANDLED_EXCEPTION_TYPE(static_cast<ExceptionType>(ExceptionType_OutOfMemory | ExceptionType_StackOverflow));
 
-    ServerThreadContext * contextInfo = HeapNew(ServerThreadContext, threadContextData);
+    ServerThreadContext * contextInfo = HeapNewNoThrow(ServerThreadContext, threadContextData);
```

Comment:
are you sure? I thought i have seen it fire before. but i guess my point was we should probably have try/catch or use no throw allocator

---
In reply to: [81830381](https://github.com/Microsoft/ChakraCore/pull/1689#discussion_r81830381) [](ancestors = 81830381,81828161,81825163)